spec/shared/db/schema.rb

```diff
@@ -42,7 +42,7 @@
 
   create_table "votes", id: :serial, force: :cascade do |t|
     t.integer "voteable_id", null: false
-    t.string "voteable_type", null: false
+    t.string "voteable_type", limit: 255, null: false
```

Comment:
@zdennis is there a migration that needs to get checked in as part of this?

Reply:
There is not. This change is from `rake db:schema:dump` from Rails 5.

I ran `rake db:schema:dump` in Rails 3 and Rails 4 and they both have a lot more changes so I assumed the schema was made from a Rails 5 dump. 

I can remove this change from the schema. I'd be interested to find out if you dump it locally if it adds it back in (is it a postgres version thing affecting schema dumping?)

Reply:
_yikes_ lol i have no recollection of how/where i generated this schema dump but i'm surprised and concerned that it's different between rails versions

in any case probably not worth checking in these changes as part of this PR?

Reply:
I've rebased out the  `limit: 255` change on the `voteable_type` column.